tests/integration/targets/wafv2/tasks/create_webacl.yml

```diff
@@ -88,6 +92,25 @@
           managed_rule_group_statement:
             vendor_name: AWS
             name: AWSManagedRulesAdminProtectionRuleSet
+      - name: rate-limit-per-IP
+        priority: 3
+        action:
+          block:
+            custom_response:
+              response_code: 429
+              custom_response_body_key: too_many_requests
+        statement:
+          rate_based_statement:
+            limit: 5000
+            aggregate_key_type: IP
+          visibility_config:
+            sampled_requests_enabled: yes
+            cloud_watch_metrics_enabled: yes
+            metric_name: waf-acl-rule-rate-limit-per-IP
+    custom_response_bodies:
+      too_many_requests:
+        Content-Type: APPLICATION_JSON
```

Comment:
I think this exceptions of keynames must be expanded here: https://github.com/ansible-collections/community.aws/blob/main/plugins/module_utils/wafv2.py#L48